nautobot/core/templates/components/panel/header_table.html

```diff
@@ -0,0 +1,3 @@
+{% load helpers %}
+
+<strong>{{ body_content_table_verbose_name_plural|bettertitle }}</strong> <span class="badge badge-default">{{ body_content_table.rows|length }}</span>
```

Comment:
Sooner or later we'll probably want an option to specify a custom title for one of these panel tables. Maybe not essential at this moment, but maybe have the `verbose_name_plural|bettertitle` be done in code instead of in the template?